zdns/main.go

```diff
@@ -93,6 +95,24 @@ func main() {
 	// complete post facto global initialization based on command line arguments
 	gc.Timeout = time.Duration(time.Second * time.Duration(*timeout))
 	gc.IterationTimeout = time.Duration(time.Second * time.Duration(*iterationTimeout))
+	// class initialization
+	switch *class_string {
```

Comment:
It would be nice to do a `strings.ToUpper()` here. Usability wise, it's a bit friendlier to also allow lowercase values.